syncs/github-dependabot/main.ts

```diff
@@ -41,12 +42,14 @@ const octokit = new OctokitWithThrottling({
       },
 });
 const alertsBuffer = [];
+const perPage = params.perPage || 100;
 
 const iterator = octokit.paginate.iterator(`GET /repos/${owner}/${repo}/dependabot/alerts`, {
     headers: {
         'X-GitHub-Api-Version': '2022-11-28'
     },
-    owner, repo
+    owner, repo,
+    per_page: perPage,
```

Comment:
See: https://docs.github.com/en/rest/dependabot/alerts?apiVersion=2022-11-28#list-dependabot-alerts-for-a-repository--parameters